components/FlyOut/DualFlyOut.vue

```diff
@@ -0,0 +1,91 @@
+<script lang="ts">
+import { defineComponent } from "vue";
+export default defineComponent({
+  name: "DualFlyOut",
+  props: {
+    primaryVisibility: {
+      type: Boolean,
+      default: false
+    },
+    secondaryVisibility: {
+      type: Boolean,
+      default: false
+    }
+  },
+  emits: ["primaryChange", "secondaryChange"],
+  data () {
+      return {
+          primaryOpen: this.primaryVisibility,
+          secondaryOpen: this.secondaryVisibility,
+          windowWidth: 0
+      };
+  },
+  mounted () {
+    this.windowWidth = window.innerHeight;
```

Comment:
This should be
```js
this.windowWidth = window.innerWidth;
```
Because else, the dual-flyout menu will not be able to open up both panels until you resize, even if you have a wide enough screen at the beginning.

Reply:
Yep, you're correct, sorry xD